requirements/dev.txt

```diff
@@ -25,17 +25,5 @@ pre-commit==1.10.5
 # Django SSL for local https development
 django-sslserver==0.19
 
-# Django Debug Toolbar for local development
-django-debug-toolbar==1.6
-
-# Ipdb
-ipdb==0.10.1
-
 # PyDevD (Remote Debugging)
 pydevd==0.0.6
-
-# n+1 query detection
-nplusone==0.8.2
```

Comment:
Ditto, probably safe to remove, but check with team as well. In addition, we want to document these library removals